src/strategies/mantra-dao/README.md

```diff
@@ -0,0 +1,19 @@
+MANTRA DAO strategy
+
+The strategy combines balances of two different staking contracts and outputs a merged score.
+
+The ANTRA DAO strategy gives the option for one of the staked balances scores to be scaled up or down in terms of vote weighting.
+passing -> options.omUniScalingFactor
+
+Our tokens are the OM erc20 and the UNI V2 OM synthetic token found at:
+omStakingAddress and omUniStakingAddress
+
+Example:
+
+A address has a staked balance of 10 in contract-1 and a staked balance of 10 in contract-2.
+
+with a scaling factor of 1.0 their output score will be 20 (10 + 10)
+with a scaling factor of 2.0 their output score will be 30 (10 + 20)
```

Comment:
Sorry i think this is something you can do already with another strategy called "erc20-balance-of-coeff". Here is an example of config you can use:
```json
{
  "strategies": [
    {
      "name": "erc20-balance-of",
      "params": {
        "address": "0x2bCD929283Ad0EE603E743412ddb214b91fbab88",
        "symbol": "OM",
        "decimals": 18
      }
    },
    {
      "name": "erc20-balance-of-coeff",
      "params": {
        "address": "0x99b1db3318aa3040f336fb65c55400e164ddcd7f",
        "symbol": "OMUNI",
        "decimals": 18,
        "coeff": 2
      }
    }
  ]
}
```
Can you check to see if you can use that strategy?

Reply:
Ok good suggestion.

Our problem with using that is, this only a first iteration, and were likely adding more complexity as time goes on, so we were hoping to have our own strategy that we can change.

Reply:
Ok perhaps we can use this one for now to get us going and propose our own when we alter it. Appreciate the advice!